erpnext/hr/doctype/employee/employee.json

```diff
@@ -55,6 +55,7 @@
   "leave_policy",
   "attendance_device_id",
   "leave_approver",
+  "expense_approver",
```

Comment:
this shouldnt be in leave details section, maybe shift it to salary details since it is fiscally related